core-build/gulp-core-build-sass/src/CSSModules.ts

```diff
@@ -0,0 +1,57 @@
+import * as path from 'path';
+
+import * as postcss from 'postcss';
+import * as cssModules from 'postcss-modules';
+import * as crypto from 'crypto';
+
+export interface ICSSModules {
+  /**
+   * Return a configured postcss plugin that will map class names to a
+   * consistently generated scoped name.
+   */
+  getPlugin: () => postcss.AcceptedPlugin;
+
+  /**
+   * Return the CSS class map that is stored after postcss-modules runs.
+   */
+  getCssJSON: () => Object;
```

Comment:
`JSON` should be `Json`. Our convention is to only UPPERCASE acronyms shorter than three characters.

Reply:
Good to know. After I slept on it, I realized that the name is kind of wrong, so I changed it.